src/Microsoft.ComponentDetection.Contracts/TypedComponent/NpmComponent.cs

```diff
@@ -29,5 +29,5 @@ public NpmComponent(string name, string version, string hash = null, NpmAuthor a
 
     public override string Id => $"{this.Name} {this.Version} - {this.Type}";
 
-    public override PackageURL PackageUrl => new PackageURL("npm", null, this.Name, this.Version, null, null);
+    public override PackageURL PackageUrl => new PackageURL("npm", null, this.Name.ToLower(), this.Version, null, null);
```

Comment:
This doesn't match the PURL implementation. They rely on `npmjs.com` enforcing the lowercase package name: https://github.com/package-url/packageurl-dotnet/commit/810a77143131a076273ce7ce0c48f7168ff44156